<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SafeUTool</class>
 <widget class="QMainWindow" name="SafeUTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>隐藏区管理工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbGetCap">
        <property name="text">
         <string>打开磁盘</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbReadSector">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>读取扇区</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>开始扇区号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leStartSector">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>读取扇区数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSectorNumber">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbMax">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="teData"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbPartition">
        <property name="text">
         <string>分区</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCryptoConfig">
        <property name="text">
         <string>加密设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>316</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbPartition</sender>
   <signal>clicked()</signal>
   <receiver>SafeUTool</receiver>
   <slot>sltPartition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCryptoConfig</sender>
   <signal>clicked()</signal>
   <receiver>SafeUTool</receiver>
   <slot>sltCryptoConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbGetCap</sender>
   <signal>clicked()</signal>
   <receiver>SafeUTool</receiver>
   <slot>sltGetCap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbReadSector</sender>
   <signal>clicked()</signal>
   <receiver>SafeUTool</receiver>
   <slot>sltReadSector()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltPartition()</slot>
  <slot>sltGetCap()</slot>
  <slot>sltReadSector()</slot>
  <slot>sltCryptoConfig()</slot>
 </slots>
</ui>
